sync-service/latest/support/index.md

```diff
@@ -6,27 +6,27 @@ The following are the supported platforms for Alfresco Sync Service 3.8:
 
 | Version | Notes |
 | ------- | ----- |
+| Alfresco Content Services 7.3 | Optionally with Alfresco Governance Services 7.3 |
 | Alfresco Content Services 7.2 | Optionally with Alfresco Governance Services 7.2 |
-| Alfresco Content Services 7.1 | Optionally with Alfresco Governance Services 7.1 |
-| Alfresco Content Services 7.0 | Optionally with Alfresco Governance Services 3.5 |
-| Identity Service 1.4 or later | Required for SAML authentication |
-| Alfresco Desktop Sync for Windows 1.7 or later | |
-| Alfresco Desktop Sync for Mac 1.7 or later | |
+| Identity Service 1.8 or later | Required for SAML authentication |
+| Alfresco Desktop Sync for Windows 1.13.1 or later | |
+| Alfresco Desktop Sync for Mac 1.13.1 or later | |
 | | |
-| **Java** |
+| | Check the [Alfresco Content Services Supported platforms]({% link content-services/latest/support/index.md %}) page for specific versions of the individual components. |
+| **Java** | |
+| OpenJDK 17 (64-bit) | |
 | OpenJDK 11 (64-bit) | |
-| Oracle Java 8 onwards | |
 | | |
 | **Message brokers** |
-| Apache ActiveMQ 5.15.8 or later | |
+| Apache ActiveMQ | |
```

Comment:
@anxumalo any specific reasons why we have removed versions for the applications?

Reply:
Hi @abhishekkumar31, the main reason is because Sync Service relies on the same components as ACS. I also mentioned this in the JIRA ticket: "For easier maintenance, I’ll remove the version numbers of the various components and direct users to the ACS Supported platforms."

Reply:
Thanks for this info @anxumalo